.appveyor/build_python.bat

```diff
@@ -0,0 +1,229 @@
+
+cd PC
+nmake /f icons.mak
+if not %ERRORLEVEL% == 0 (
+    echo "Failed to build icons for Python 32bit"
+    exit /b 1
+)
+
+cd "%BUILDDIR%\cpython-2.7.13\Tools\msi"
+set PCBUILD=PC\VS9.0
+set SNAPSHOT=0
+"%BUILDDIR%\cpython-2.7.13\%PCBUILD%\python.exe" -m ensurepip -U --default-pip
+if not %ERRORLEVEL% == 0 (
+    echo "Failed to run ensurepip for Python 32bit"
+    exit /b 1
+)
+"%BUILDDIR%\cpython-2.7.13\%PCBUILD%\python.exe" -m pip install pypiwin32
+if not %ERRORLEVEL% == 0 (
+    echo "Failed to install pypiwin32 for Python 32bit"
+    exit /b 1
+)
+"%BUILDDIR%\cpython-2.7.13\%PCBUILD%\python.exe" msi.py
+if not exist "%BUILDDIR%\cpython-2.7.13\Tools\msi\python-2.7.13150.msi" (
+    echo "Failed to generate msi Python 32bit"
+    exit /b 1
+)
+
+start /wait msiexec /qn /a "%BUILDDIR%\cpython-2.7.13\Tools\msi\python-2.7.13150.msi" TARGETDIR="%BINARIESDIR%\python"
+if not %ERRORLEVEL% == 0 (
+    echo "Failed to extract msi Python 32bit to %BINARIESDIR%\python"
+    exit /b 1
+)
+"%BINARIESDIR%\python\python.exe" -Wi "%BINARIESDIR%\python\Lib\compileall.py" -q -f -x "bad_coding|badsyntax|site-packages|py3_" "%BINARIESDIR%\python\Lib"
+"%BINARIESDIR%\python\python.exe" -O -Wi "%BINARIESDIR%\python\Lib\compileall.py" -q -f -x "bad_coding|badsyntax|site-packages|py3_" "%BINARIESDIR%\python\Lib"
+"%BINARIESDIR%\python\python.exe" -c "import lib2to3.pygram, lib2to3.patcomp;lib2to3.patcomp.PatternCompiler()"
+"%BINARIESDIR%\python\python.exe" -m ensurepip -U --default-pip
+if not %ERRORLEVEL% == 0 (
+    echo "Failed to run ensurepip in %BINARIESDIR%\python for Python 32bit"
+    exit /b 1
+)
+%BINARIESDIR%\python\python.exe -m pip install pypiwin32
+if not %ERRORLEVEL% == 0 (
+    echo "Failed to install pypiwin32 in %BINARIESDIR%\python for Python 32bit"
+    exit /b 1
+)
+copy /B /Y "%VS90COMNTOOLS%..\..\VC\redist\amd64\Microsoft.VC90.CRT\msvcr90.dll" "%BINARIESDIR%\python\"
+if not %ERRORLEVEL% == 0 (
+    echo "Failed to copy msvcr90.dll in %BINARIESDIR%\python for Python 32bit"
+    exit /b 1
+)
+copy /B /Y "%VS90COMNTOOLS%..\..\VC\redist\amd64\Microsoft.VC90.CRT\Microsoft.VC90.CRT.manifest" "%BINARIESDIR%\python\"
+if not %ERRORLEVEL% == 0 (
+    echo "Failed to copy Microsoft.VC90.CRT.manifest in %BINARIESDIR%\python for Python 32bit"
+    exit /b 1
+)
+
+
+REM Start of build for Python 64 bit
+cd "%BUILDDIR%"
+
+if exist "%BINARIESDIR%\python_x64" (
+    echo "%BINARIESDIR%\python_x64 exist already!"
+    exit /b 1
+)
+
+2>nul rd /S /Q "%BUILDDIR%\cpython-2.7.13"
+"%UNZIP_BIN%" -q "%BUILDDIR%\cpython-2.7.13.zip"
+cd "%BUILDDIR%\cpython-2.7.13"
+
+REM Restore externals directory if python_externals.tar.gz exists
+if exist "%BUILDDIR%\python_externals.tar.gz" (
+    if not exist "%BUILDDIR%\cpython-2.7.13\externals" (
+        "%MSYSDIR%\bin\bash" --login -i -c "cd $BUILDDIR_M/cpython-2.7.13 && tar -xf $BUILDDIR_M/python_externals.tar.gz"
+    )
+) else (
+	call "%BUILDDIR%\cpython-2.7.13\PCbuild\get_externals.bat"
```

Comment:
No, because PC\VS9.0\build.bat with call it if you pass '-e' argument to build.bat, I am calling here specifically because we have to delete one file (externals\openssl-1.0.2j\ms\nt64.mak) from openssl directory as there is bug (https://bugs.python.org/issue30368) in Python 2.7.13 source code, without deleting this file it won't build ssl module in 64 bit.